cmd/tsbs_generate_queries/utils/time_interval.go

```diff
@@ -0,0 +1,76 @@
+package utils
```

Comment:
this file was moved to `internal/utils/`, so you'll need to remove this and update imports

Reply:
slipped out of my sight, removing it